calico-enterprise/getting-started/bare-metal/troubleshoot.mdx

```diff
@@ -0,0 +1,111 @@
+---
+description: Troubleshoot non-cluster hosts and VMs setup
+---
+
+# Troubleshoot non-cluster hosts and VMs setup
```

Comment:
The article is located in the installation section. Is this the right place? 

I wonder whether it's better placed in the troubleshooting section: https://docs.tigera.io/calico-enterprise/latest/operations/troubleshoot/.